packages/frontend/src/view/pages/forced-actions/NewSpotForcedWithdrawalPage.tsx

```diff
@@ -0,0 +1,119 @@
+    <>
+      Once your request has been processed the status will change to{' '}
+      <span className="text-yellow-300">’processed’</span> and you will be able
+      to withdraw your funds by submitting a withdrawal transaction.
+    </>,
+  ]
+  const assetInfo = assetToInfo({ hashOrId: props.asset.hashOrId })
+  const propsJson = serializeForcedActionsFormProps(props)
+  const formattedBalance = formatAmount(
+    { hashOrId: props.asset.hashOrId },
+    props.asset.balance
+  )
+  return (
+    <Page
+      path="/forced/new/:positionId/:assetId"
+      description="Page that allows user withdrawal of spot asset"
+      user={props.user}
+    >
+      <main className="mx-auto flex-1 p-16">
+        <div className="my-auto flex gap-12">
+          <div className="mt-6 flex flex-col">
+            <span className="text-xl font-semibold">
+              Begin withdrawal process
+            </span>
+            <span className="mt-6 text-sm font-semibold text-zinc-500">
+              The withdrawal process consists of three steps:
+            </span>
+            <OrderedList items={instructions} className="mt-3 max-w-md" />
+          </div>
+          <Card className="h-min w-[480px]">
+            <form
+              id={SpotWithdrawalFormId}
+              className="flex flex-col gap-6"
+              data-props={propsJson}
+            >
+              <div className="flex items-end justify-between">
+                <span className="text-xl font-semibold">Withdrawal</span>
+                <span>
+                  <span className="text-sm font-medium text-zinc-500">
+                    Vault
+                  </span>{' '}
+                  <span className="text-lg font-semibold">
+                    #{props.positionOrVaultId.toString()}
+                  </span>
+                </span>
+              </div>
+              <div className="flex flex-col gap-2">
+                <ForcedActionCard>
+                  <div className="flex gap-2">
+                    <div className="flex flex-1 flex-col gap-2">
+                      <span className="text-sm font-medium text-zinc-500">
+                        Balance
+                      </span>
+                      <span className="text-xl font-semibold">
+                        {formattedBalance}
+                      </span>
+                    </div>
+                    <div className="flex flex-col items-end gap-2">
+                      <span className="text-sm font-medium text-zinc-500">
+                        Asset
+                      </span>
+                      <AssetWithLogo assetInfo={assetInfo} />
+                    </div>
+                  </div>
+                </ForcedActionCard>
+              </div>
+              <div className="flex flex-col gap-2">
+                <Button className="w-full">Prepare for withdrawal</Button>
```

Comment:
This button does nothing onClick. Should that be the case?

Reply:
Button by default submits form if it is in form. And submit is handled in scripts.